css/details.css

```diff
@@ -1,10 +1,22 @@
 table {
-	display: inline-block;
+	display: table;
 	width: 48%;
+	float: left;
+	margin-bottom: 1.5em;
+}
+
+@media(max-width: 1024px) {
+	table {
+		width: 80%;
+	}
 }
 
 caption {
 	font-weight: bold;
+	border-bottom: 1px solid #C3C3C3;
+	margin-left: 20%;
+	margin-bottom: 1em;
+	font-size: 115%;
```

Comment:
`<caption>` already represents a title. In terms of style, h3 would save up setting font-size, but not the other attributes.